### ISSUE TITLE
The `aria-label` cannot be customized.

### ISSUE THREAD

**Issue by hirotomoyamada** [emilkowalski/sonner#702]
Currently, the `aria-label` for `section` is set to `${containerAriaLabel} ${hotkeyLabel}`, preventing developers from setting an arbitrary `aria-label`.

**Comment by David-Botos** [comment]
What versioned package release do we expect to get access to this commit?
